src/lemon/cbc.cc

```diff
@@ -21,6 +21,7 @@
 
 #include "cbc.h"
 
+#ifdef LEMON_HAVE_CBC
```

Comment:
Remove this and other added `#ifdef`/`#endif`. We will not compile this file. AFAICT, we need only `base.cc` and `random.cc`.